docs/how-to/redeploy.md

```diff
@@ -0,0 +1,19 @@
+# How to redeploy Jenkins
+
+We consider a redeployment is the process where the old charm instance is removed and data is migrated to a new charm instance.
+
+As Jenkins does not use any persistent storage or database, redeploying the charm consists of 3 steps:
+
+1. Create the new Jenkins charm instance
+```bash
+juju deploy jenkins-k8s jenkins-k8s-new
+```
+2. Migrate Jenkins data
+See the `Migrate Jenkins data` section below.
+3. Remove the old Jenkins charm instance
+```bash
+juju remove-application jenkins-k8s
+```
+
+### Migrate Jenkins data
+Follow the instructions in [the charm's documentation for resizing storage](https://charmhub.io/jenkins-k8s/docs/how-to-resize-jenkins-storage) to migrate the data to the new Jenkins charm instance.
```

Comment:
As discussed on a call just now, let's move the "backup and restore" parts of that doc to its own how-to and then refer to that from here and also from the how-to-resize-jenkins-storage. We already have a ticket for creating an action for those steps as well, but that's outside of the scope of this change.

Reply:
will do, thanks, this will indeed make it more straightforward

Reply:
I updated the docs to reflect this